feat: improve HTML semantics and buttons animation

SebastianL-dev opened the pull request:
- Improved HTML semantics and fix buttons hover transition.

SebastianL-dev commented:
Hey, the design its the same, i just fixed some HTML semantics, and the buttons transitions its better, before all buttons have no transitions, now have fade transition on hover.


https://github.com/user-attachments/assets/71b0444e-2076-4c98-8fcd-f98de9140e08

It's a little change but i think that look much better.

mauro-balades commented:
Doesn't it break the initial animations though?

SebastianL-dev commented:
Nope, it doesn't, you can see it in the video 😊

ConwayTech-Dev commented:
Hey, nice work! This change seems fine for the most part, but it does break the color of the "Start Exploring" button. On the main version, it's dark (to contrast it from the "Download" button), but on your version, it's light. Shouldn't be too hard to fix.

There might be some other color issues too, but that's the main one that I noticed.

SebastianL-dev commented:
Hey, sorry for that, i didn't realize about that, i will fix it and make another pull request, thank you. 😊

SebastianL-dev commented:
- Fixed buttons transitions, now looks better and don't break the contrast in buttons. 


https://github.com/user-attachments/assets/3e37af8d-e969-476b-9a51-766140e5d240

Hey, i fixed the transitions, now the transitions looks better and it doesn't break the contrast.
